3rdparty/obiwarp/obiwarp.h

```diff
@@ -0,0 +1,46 @@
+#ifndef OBIWARP_H
+#define OBIWARP_H
+
+#include <vector>
+#include <iostream>
+#include <assert.h>
+
+#include "vec.h"
+#include "mat.h"
+#include "dynprog.h"
+
+using namespace std;
+
+class ObiWarp{
+public:
+    ObiWarp(string score="cor",bool local = false, float factor_diag = 2.f, float factor_gap = 1.f, 
```

Comment:
Should we not use a`struct` here ? 
You have added a struct ,`obiParams`, in `mzAligner.cpp` , can we not pass that struct here ?